### PR TITLE
HTBHF-624 Added acceptance tests for generic error page.

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: src/test/acceptance/features/error-page.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: Sever Error page
```
Comment (gethyn1) [inline src/test/acceptance/features/error-page.feature]: typo - I guess this should be 'Server'

File: src/test/acceptance/features/error-page.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  In order to facilitate the beta HTBHF programme
```
Comment (gethyn1) [inline src/test/acceptance/features/error-page.feature]: Should we be using `beta`? I'm assuming we will still be using this code once the beta is complete?

File: src/test/acceptance/features/error-page.feature (hunk 1 above)
Comment (dwybourn) [reply src/test/acceptance/features/error-page.feature]: Fixed typo.

File: src/test/acceptance/features/error-page.feature (hunk 2 above)
Comment (dwybourn) [reply src/test/acceptance/features/error-page.feature]: We have this in most of the feature files. I would leave it in for now as the program is beta.

**Pull request merged**